builtin/k8s/platform.go

```diff
@@ -1475,8 +1475,8 @@ type Config struct {
 // ResourceConfig describes the request and limit of a resource. Used for
 // cpu and memory resource configuration.
 type ResourceConfig struct {
-	Request string `hcl:"request,optional" json:"request"`
-	Limit   string `hcl:"limit,optional" json:"limit"`
+	Request string `hcl:"Requested,optional" json:"Requested,optional"`
+	Limit   string `hcl:"Limit,optional" json:"Limit,optional"`
```

Comment:
Hm, I don't think this change is right. It should be how it originally is. Not sure if it needs `json,limit,optional` though.

Reply:
Yea, I think this PR maybe should be closed. I shared some logs in https://github.com/hashicorp/waypoint/issues/3534, and this PR happened help me get past an erroring `waypoint up`, but I'm certainly missing some broader understanding 😬.

Reply:
Ok yeah, I'd close this one then! I saw your message on the other issue, feel free to continue the conversation there.